samples/subsys/canbus/canopen/README.rst

```diff
@@ -61,6 +61,12 @@ The sample can be built and executed for the FRDM-K64F as follows:
 Pressing the button labelled ``SW3`` will increment the button press
 counter object at index ``0x2102`` in the object dictionary.
 
+Building and Running for other boards
+=====================================
+This project can be built for other boards same way as for :ref:`frdm_k64f` board.
+Although not all boards have internal storage. In this case this project should be 
+build with `-DCONF_FILE=prj_no_storage.conf` additional config file.
```

Comment:
Could we present this as a `.. zephyr-app-commands::` section as used for the other instructions?

Reply:
Done